libs/RobotKit/include/RobotController.h

```diff
@@ -191,6 +193,18 @@ class RobotController : public interface::RobotController
 
 	void startDisconnectionBehavior() final { stopActuators(); }
 
+	void startAutonomousActivity() final
+	{
+		_lcd.turnOn();
+		_behaviorkit.chooseActivity();
+	}
+
+	void stopAutonomousActivity() final
```

Comment:
pareil dans l'autre sens

```suggestion
	void stopAutonomousActivitiesMode() final
```